cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -470,6 +470,8 @@ public SessionListener initSessionListener() {
 
                 if (this.appBannerModule != null && this.getCurrentActivity() != null) {
                     this.appBannerModule.initSession(channelId);
+                } else {
```

Comment:
change else to else if(this.getCurrentActivity() == null)
or just if(this.getCurrentActivity() == null)

Reply:
and change
"activity null: " + getCurrentActivity()
to 
"getCurrentActivity() is null"

Reply:
```
                if (this.appBannerModule != null && this.getCurrentActivity() != null) {
                    this.appBannerModule.initSession(channelId);
                } else if(this.getCurrentActivity() == null){
                    Logger.e("LOG_TAG", "getCurrentActivity() is null");
                }

```